tests/unit/maxmemory.tcl

```diff
@@ -415,17 +415,33 @@ test_slave_buffers {slave buffer are counted correctly} 1000000 10 0 1
 test_slave_buffers "replica buffer don't induce eviction" 100000 100 1 0
 
 start_server {tags {"maxmemory external:skip"}} {
-    test {Don't rehash if used memory exceeds maxmemory after rehash} {
+    test {Don't resize if used memory exceeds maxmemory after resize} {
```

Comment:
i rather not change the name of the test if the old one is also valid.
maybe someone has it on his skipfile